configure.ac

```diff
@@ -128,6 +128,7 @@ if test "x$exec_prefix" = "xNONE"; then
 fi
 pidfiledir='${localstatedir}'
 moduledir='${exec_prefix}/lib/syslog-ng'
+loggenplugindir='${exec_prefix}/lib/syslog-ng/loggen'
```

Comment:
The implementation of loggen is more or less independent from the syslog-ng (less because compat layer is in loggen as well).
Therefore I don't want to move it to /lib/loggen.